app/components/ui/Navigation.tsx

```diff
@@ -142,54 +131,3 @@ const SiteLinks = () => {
     </ul>
   );
 };
-
-const SiteSearch = ({
```

Comment:
Moved to https://github.com/Exygy/askdarcel-web/pull/78/files#diff-0fe3853dad019189d835c6ef43b97dbd26e1cc1cd26cdcec5f924d7c625d3823R14